components/Slider.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import "react-responsive-carousel/lib/styles/carousel.min.css";
+import { Carousel } from 'react-responsive-carousel';
+
+export default class Hazhar extends Component {
```

Comment:
- Pay attention to your component naming, change ```Hazhar``` to ```Slider```.

Reply:
The problem was solved and the tags changed